src/services/youtube/youtube.js

```diff
@@ -1,8 +1,26 @@
-import * as mockData from '../../utils/youtube-videos-mock.json';
+import { fetchData } from '../fetch-data';
 
-export const getYoutubeMainInfoList = () => {
-  const mainInfo = mockData.items.map((item) => ({
-    id: item.etag,
+export const getYoutubeMainInfoList = async (search = 'Wizeline') => {
+  const url = `https://youtube.googleapis.com/youtube/v3/search?part=snippet&maxResults=18&q=${encodeURI(
+    search
+  )}&type=video&key=AIzaSyBYzWLlzPQfofKh32frvUh6oKJgB98ghis`;
```

Comment:
API_KEY is a sensitive data, and you should ALWAYS handle it with a .env file: 
https://create-react-app.dev/docs/adding-custom-environment-variables/